internal/commands/push/command_test.go

```diff
@@ -1153,6 +1153,70 @@ func TestPushHandlerCreateNewApp(t *testing.T) {
 			assert.True(t, os.IsNotExist(fileErr), "expected nested config path to not exist, but err was: %s", err)
 		})
 	}
+
+	t.Run("should skip diff when creating a new app", func(t *testing.T) {
```

Comment:
I'm am skeptical about the need for this test, this is probably related to my earlier question about the extra `if !createNewApp` check around the existing confirm diffs prompt

Reply:
Yeah, resolved per our discussion. I've added testing paths for when the user receives a diff and asserting on both the output of the CLI as well as that the config file is created regardless of the diff.